lib/webview-helpers.js

```diff
@@ -289,6 +289,31 @@ helpers.procFromWebview = async function procFromWebview (adb, webview) {
   return pkg;
 };
 
+/**
+ * Retrieves webview names for getContexts
+ *
+ * @param {object} adb - an ADB instance
+ * @param {GetWebviewOpts} opts See note on getWebViewsMapping
+ *
+ * @return {Array.<string>} - a list of webview names
+ */
+helpers.getWebviews = async function getWebviews (adb, {
+  androidDeviceSocket = null,
+  ensureWebviewsHavePages = true,
+  webviewDevtoolsPort = null,
+  enableWebviewDetailsCollection = null,
+} = {}) {
+  const webviewsMapping = await this.getWebViewsMapping(adb, {
+    androidDeviceSocket,
+    ensureWebviewsHavePages,
+    webviewDevtoolsPort,
+    enableWebviewDetailsCollection
+  });
+  const result = webviewsMapping.filter(function (w) {return w.webviewName;}).map(function (w) {return w.webviewName;});
```

Comment:
```
const result = webviewsMapping.reduce((acc, {webviewName}) => {
  if (webviewName) {
    acc.push(webviewName);
  }
  return acc;
}, []);
```
or 
```
const result = webviewsMapping.filter(({webviewName}) => !!webviewName).map(({webviewName}) => webviewName);
```

Reply:
I have addressed this along with https://github.com/appium/appium-android-driver/pull/662#discussion_r492533011